### DISCUSSION TITLE
Update script ideas

### DISCUSSION THREAD

**Discussion by jmb** [edent/SuperTinyIcons#768] [General]
I've been playing with automatically generating the reference markdown file. I've update my update-readme.py script in this branch: https://github.com/jmb/SuperTinyIcons/blob/script-update/ 

At the moment it looks for same-named reference files (and I renamed a lot in my branch), then the old index.md file for any other file names and the reference url. I'm saving to REFERENCE.md for now and it could be updated to read that file in instead of the index.md (which seems rather hidden away).

I was also thinking it might be useful to generate an html page with the `border-radius: 50%;` css to check that the logo fits in a circle. You can't do this in markdown I don't think as the style attribute just gets removed.

Any thoughts?

**Comment by edent** [comment]
Do it!

**Comment by jmb** [comment]
Awesome, I've made a first bash at it: #770
